lib/order.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'menu'
+require_relative 'text'
+class Order
+  attr_reader :current_order, :order_total
+
+  def initialize
+    @current_order = []
+  end
+
+  def select_item(menu, item, quantity)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)